### DISCUSSION TITLE
Using with a custom Outputs folder

### DISCUSSION THREAD

**Discussion by Neuralsplyce** [toutjavascript/Fooocus-Log-Viewer#8] [Q&A] [answered]
I modified the Fooocus config.txt file to set the outputs folder to another drive. How can I configure/edit Log Viewer to use that folder instead of ./Fooocus/outputs ?  I tried changing the url to localhost:7865/file=(my custom output path)  

I get this error: 
"detail": "File not allowed: D:/Users/Pictures/Stable_Diffusion_Images/Fooocus/viewer.html."

Thanks

**Comment by toutjavascript** [comment]
Hi
I will try that today. 
I hope it will be possible to configure the tool in that situation

Thanks for writing

**Comment by toutjavascript** [comment]
So I try with a different output folder.
Even with Fooocus tools, preview is broken
When you clic on history log here,
![Capture d’écran 2024-01-25 à 12 55 18](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/30899600/ee98c82c-97f3-4d46-a204-412e5725953d)

You got
![Capture d’écran 2024-01-25 à 12 55 29](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/30899600/1dfe720d-f696-418f-94ca-a79697f591f0)

The pictures displayed during generation come from a temp gradio folder:
![Capture d’écran 2024-01-25 à 12 57 25](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/30899600/79ee0e52-dc3a-4463-a75c-9d552742d6dc)

I am afraid the tool cannot work with external output folder...

**Comment by toutjavascript** [comment, marked as answer]
I managed to get the Log Viewer good even when outputs folder is on an other drive 👍 

Installation is a little bit more complicated:

1. You should delete the outputs folder on Fooocus directory installation
2. Then run that MKLINK command to create a symbolic junction between the Fooocus outputs standard folder to your new one (Windows command)
`mklink /J  C:\Users\XXXXX\Desktop\IA\Fooocus\Fooocus\outputs D:\pictures\outputs`
3. Open the log viewer with the full link
`http://127.0.0.1:7865/file=C:/Users/XXXXX/Desktop/IA/Fooocus/Fooocus/outputs/viewer.html`

It seems search box is broken. But I will correct it on next release.

Thanks to challenge me

**Reply by Benm678** [reply]
mklink only works with local volumes, in my case outputs are written to a network share. Is it possible to make it work with the network drive or not? 

Ok, I got it to work. 
open CMD as admin
mklink /D F:\Fooocus_win64_2-1-791\Fooocus\outputs \\SERVER\SHARE\Fooocus\Outputs

As expected it is a little slower due to the network, but works fine - thanks for your work!

**Reply by toutjavascript** [reply]
Thank you for this share
